### PR TITLE
Fix event size restriction

### PR DESCRIPTION
**Pull request by turt2live**
Fixes https://github.com/matrix-org/matrix-doc/issues/3126

### CODE REVIEW

File: content/client-server-api/_index.md [hunk 1]
```diff
@@ -1447,7 +1447,7 @@ following fields.
 
 ### Size limits
 
-The complete event MUST NOT be larger than 65535 bytes, when formatted
+The complete event MUST NOT be larger than 65536 bytes, when formatted
```
Comment (richvdh) [inline content/client-server-api/_index.md:1450]: any chance I could be a pain and ask you to change "65KB" a few lines below to `64KiB`, while you're in the area?

File: changelogs/client_server/newsfragments/3127.clarification [hunk 2]
```diff
@@ -0,0 +1 @@
+Fix various typos throughout the specification.
```
Comment (richvdh) [inline changelogs/client_server/newsfragments/3127.clarification:1]: I kinda feel like we should call this out specifically, for the conduits of this world.

**Review by richvdh — APPROVED** [review]

**Comment by kevincox** [comment]
It may be the wrong spot for this but 65535 seems like a better size because you can store it in a 16bit integer. 65536 is annoyingly one too large.

**Comment by turt2live** [comment]
@kevincox that may be, but this is fixing a bug in the spec. Making it 65535 would require an MSC.

**Pull request merged**